src/plugins/core/range.ts

```diff
@@ -401,6 +404,19 @@ export class RangeAdapter implements CommandHandler<CoreCommand> {
     return `${prefixSheet ? sheetName + "!" : ""}${rangeString}`;
   }
 
+  /** Same as getRangeString but return undefined for invalid ranges */
+  getValidRangeString(
```

Comment:
it's not the same as getRangeString, you allow to pass no range at all. you should follow the same signature don't you think?

Reply:
Allowing passing an undefined range is pretty useful, it removes 5+ ternaries. I thi k it's worth not havign the exact same signature for this. I can change the jsDoc tho if it bother you.

Reply:
imo, the 5 ternaries is not a valid point (again, just my pov). I think the getter should not be twisted so that it fits too much the usecase you have. By that logic, every ternary could/should be removed in favor of a function/getter that takes care of the garbage payload or corner cases. I going overboard with this but still, if I have 2 getters that pretty much do the same thing, I would expect the same payload.

Reply:
IMO I don't mind having a slighltu different signature if that means that it's easier to use. But I don't care that much and I'm not gonna fight for it, I updated it to have the exact same signature as getRangeString.